app/Http/Controllers/Multiplayer/Rooms/Playlist/ScoresController.php

```diff
@@ -193,6 +193,8 @@ public function update($roomId, $playlistId, $scoreId)
             ->where('id', $playlistId)
             ->firstOrFail();
 
+        abort_if($playlistItem->expired, 422, 'expired playlist item');
```

Comment:
This probably should be done in room's (`app/Models/Multiplayer/Room.php`) `assertValidStartPlay`